app/controllers/base_controller.rb

```diff
@@ -3,9 +3,10 @@ class BaseController < ApplicationController
 
   rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
 
-  after_action :verify_authorized, except: :index
+  after_action :verify_authorized
 
```

Comment:
I think we are showing a dashboard. That happens to contain a list of records specific to a user, so in a sense we are also not exactly presenting a Defence Requests index. Does that make sense?